### DISCUSSION TITLE
Menu bar (or keyboard shortcut) search

### DISCUSSION THREAD

**Discussion by az-pz** [Flow-Launcher/Flow.Launcher#2478] [General]
Does anyone know of any plugin where I can discover the app specific shortcuts? I moved from mac to windows. I had https://alfred.app/workflows/benziahamed/menu-bar-search/ in alfred which would show me all menu bar options with their shortcuts. That helped me learn the shortcuts and it was a nice fallback.

**Comment by taooceros** [comment]
I don't think windows has something like  that?

**Comment by Garulf** [comment]
I am familiar with that plugin. But windows doesn't allow easy access to UI elements like this. 

It is possible to do I think. But no plugin is available for this sorry.
